util/find-doc-nits

```diff
@@ -872,6 +872,22 @@ if ( $opt_c ) {
 }
 
 if ( $opt_l ) {
+    # Read the "missing" file.
+    open my $MISS, "util/missingdocs.txt"
+        or die "Can't open missingdocs.txt, $!";
+    while ( <$MISS> ) {
+        chop;
+        next unless m@man[1357]/@;
+        my $file = $_;
+        err("util/missingdocs.txt:1: File doc/$file exists")
+            if -f "doc/$file";
+        if ( $file =~ m@man([1357])/(.*).pod@ ) {
+            my $link = "$2($1)";
+            $name_map{$link} = $file;
```

Comment:
There's a problem with this; we often have several functions in one pod, so name-to-pod mapping isn't accurate.

For accuracy, it would probably be better to have a db of all documented names including the section number (should be easy to generate from the collected names coupled with the section number of the currently processed), and a db of existing links (same format), a record any name in the latter db that's missing in the former db.

Reply:
Yes, it's not perfect.  It gets things much further along however.  The issue you mention, however, can be worked-around by adding multiple entries man3/foo and man4/bar if both are in the to-be-written foo.pod page.

In my view, doing a fully accurate DB of all that information is overkill. It's also not something I think I will have the time to do.  You can consider this PR to be the first pass of that, and a later PR could turn the missing file into a api/file list.

Reply:
Another thought that occurred to me is that database will not capture the "man7" missing references (which don't have function names)

Reply:
That last remark doesn't really matter, because the links *must* match a name in a NAME section. It just so happens that man3 pages have function names there.

Reply:
Er, okay, maybe.  I still think this is a very useful step toward what you think of as the ultimate way to go.

Reply:
It would be cool if the www Makefile used this to create "this page not yet written" pages. :)

Reply:
We could have a generic 404 page that does just that...

Reply:
@levitte Could you think about this again, and see if you can live with the approach in this PR, at least for now?

Reply:
Reposting: @levitte, could you think about this again and see if you can live with the approach in this PR, at least for now?

Reply:
No.  However, I solved the problem for you, see #10621